docs/clusters.md

```diff
@@ -157,12 +157,12 @@ A History shard maintains four types of queues:
 - Replicator queue: asynchronously replicates Workflow Executions from active Clusters to other passive Clusters (experimental Multi-Cluster feature).
 - Visibility queue: pushes data to the visibility index (Elasticsearch).
 
-The History service talks to the Matching Service and the Database.
+The History Service talks to the Matching Service and the Database.
```

Comment:
We don't define "database" as being anything Temporal-specific (other than technical requirements), so unless Cully says otherwise, I say lowercase it.

Reply:
Concur. Lowercase.